trainingextension/resources/flexible-search/task3.fxs

```diff
@@ -0,0 +1,4 @@
+SELECT {a:pk}
+FROM {Address AS a
+      JOIN Customer AS c ON {a:owner} = {c:pk} }
+            WHERE {a:town} = 'London'
```

Comment:
Select all Customers that have addresses with town 'Kharkiv'. Execute from HAC. ;)